guiEditor/src/components/propertyTab/propertyTabComponent.tsx

```diff
@@ -241,12 +241,20 @@ export class PropertyTabComponent extends React.Component<IPropertyTabComponentP
     render() {
         if (this.state.currentNode) {
             return (
-                <div id="propertyTab">
-                    <div id="header">
+                <div id="propertyTab"                                 onFocus={() => {(this.props.globalState.blockKeyboardEvents = true); console.log("blocked");}}

```

Comment:
I think you should add "Remove console.log(...)" in https://github.com/BabylonJS/Babylon.js/issues/9811, else you may forget that one ;)